### PR TITLE
#652 - Verify with authentication in pypiIT

### LINKED ISSUE #652: PyPi integration tests
Add integration tests that verify that PyPi can upload and install packages with authentication enabled and disabled

### PR DESCRIPTION
**Pull request by genryxy**
Closes #652 
Verify package installation and  package publishing with authentication in `pypiITCase`

### CODE REVIEW

File: src/test/java/com/artipie/pypi/PypiITCase.java [hunk 1]
```diff
@@ -150,20 +148,130 @@ void pypiPublish() throws Exception {
...
+    private RepoConfigYaml config(final boolean anonymous) {
+        final RepoConfigYaml yaml = new RepoConfigYaml("pypi")
+            .withFileStorage(this.tmp.resolve("repos"));
+        if (!anonymous) {
+            yaml.withPermissions(
+                new RepoPerms(
+                    new RepoPermissions.PermissionItem(
+                        ArtipieServer.ALICE.name(), new ListOf<>("*")
+                    )
+                )
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/pypi/PypiITCase.java:256]: @genryxy `RepoPerms` now has ctor from two strings (name and password), so this expression can be simplified

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@genryxy just minor comment, take a look please

**Comment by genryxy** [comment]
@olenagerasimova thanks, replaced ctor

**Comment by olenagerasimova** [comment]
@genryxy thanks
@olegmoz good to merge

**Comment by olegmoz** [comment]
@olenagerasimova please approve the PR

**Review by olenagerasimova — APPROVED** [review]

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#682` is not assigned, can't get performer

**Pull request merged**